recipes/syngap/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "syngap" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yanyew/syngap_bioconda/releases/download/v{{ version }}/syngap_v{{ version }}.tar.gz
```

Comment:
please update the url to https://github.com/yanyew/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz